src/commands/indexer/index.ts

```diff
@@ -26,13 +26,17 @@ import {getIpfsCidFromTokenUri, validateIpfsCid} from '../../utils/validation'
 import {DBJob, DBJobMap} from '../../types/indexer'
 import {
   // handleMintEvent,
-  handleBridgeInEvent,
-  handleBridgeOutEvent,
-  handleContractDeployedEvent,
+  // handleBridgeInEvent,
+  // handleBridgeOutEvent,
+  // handleContractDeployedEvent,
   handleAvailableOperatorJobEvent,
 } from '../../handlers/indexer'
 
-import {handleMintEvent as sqsHandleMintEvent} from '../../handlers/sqs-indexer'
+import {
+  handleMintEvent as sqsHandleMintEvent,
```

Comment:
This is less important. But in the future you don't have to maintain the same function name. Having the original class name be `sqsHandleMintEvent` should be enough. Doing something like this can be confusing at some point.